src/api/schemas/user-schema.ts

```diff
@@ -39,6 +39,11 @@ const bio = string({
     invalid_type_error: invalidMessage('Bio', 'string')
 });
 
+const favoriteActivities = string({
```

Comment:
there's an array type validation with Zod... could try looking at that, check `/schemas/user-schemas.ts` line 57 for that.

Reply:
done

Reply:
it should be an array of number though right? i'm not sure, whichever works. what you're doing atm is array of string.